browser-tests/utils/settings.ts

```diff
@@ -7,3 +7,10 @@ export const envUrl = (): string => {
   }
   return process.env.BROWSER_TESTS_LOCAL_ENV_URL;
 };
+
+export const testUserSSN = (): string => {
+  if (!process.env.BROWSER_TESTS_SSN) {
```

Comment:
Is this already in GH secrets?

Reply:
Yes, I added it. 
I wonder if I should add it somewhere else for future generations 🤔